content/en/synthetics/api_tests/_index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: API Tests
+kind: documentation
+beta: true
+description: Simulate and monitor HTTP requests from specific locations
+aliases:
+  - /synthetics/uptime_check
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+---
+
+## Overview
+
+API tests are useful to help you monitor your API endpoints and alert you when they are failing or too slow.
+They are executed periodically from several locations around the world.
+
+Available locations are:
```

Comment:
since new locations are being added really quickly, it might be good to avoid explicitly listing them out, since we'll have to update the list very often. 

consider listing by continent instead (e.g. Asia, Australia, Europe and North America) or just pointing people to the API.

Reply:
That's a good idea!